tests/ext/reset_function_tracing.phpt

```diff
@@ -0,0 +1,32 @@
+--TEST--
+Check a function can be untraced.
```

Comment:
Small nitpick - this test could also verify other functions/methods still remain traced after one is untraced.